PurchasesTests/Purchasing/ProductsManagerTests.swift

```diff
@@ -16,15 +16,15 @@ import StoreKitTest
 @testable import RevenueCat
 import XCTest
 
-
+@available(iOS 14.0, tvOS 14.0, macOS 11.0, watchOS 6.2, *)
 class ProductsManagerTests: XCTestCase {
 
     var testSession: SKTestSession!
     var userDefaults: UserDefaults!
     var productsManager: ProductsManager!
 
     override func setUpWithError() throws {
-        testSession = try SKTestSession(configurationFileNamed: Constants.storeKitConfigFileName)
+        testSession = try SKTestSession(configurationFileNamed: "UnitTestsConfiguration")
```

Comment:
We can and should extract this, I was just being lazy, but since we need it quite a bit, I was thinking of making a `StoreKitTestCase` class that inherits from `XCTestCase`, and overrides the setup with this bit.
That way we don't have to repeat this bit in all classes that use this